src/toil/jobStores/abstractJobStore.py

```diff
@@ -224,8 +224,8 @@ def _checkJobStoreCreation(create, exists, jobStoreString):
         :raise JobStoreCreationException:  if create == exists
         """
         if create and exists:
-            raise JobStoreCreationException("The job store '%s' already exists. Use --restart or "
-                                            "'toil restart' to resume the workflow, or remove the "
+            raise JobStoreCreationException("The job store '%s' already exists. Use --restart "
```

Comment:
You might want to rewrap this string literal. I use PyCharm's `Edit —Fill Paragraph` for that. It magically works within string literals and comments. Make sure you have wrap column set to 100.